ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -185,8 +186,10 @@ const BaseFeeInput = () => {
         numeric
       />
       <AdvancedGasFeeInputSubtext
-        latest={estimatedBaseFee}
-        historical="23-359 GWEI"
+        latest={`${parseFloat(estimatedBaseFee).toFixed(2)} GWEI`}
```

Comment:
I think we should make it a habit to use BigNumber to parse WEI and GWEI instead of `parseFloat` (because there may be a lot of decimal places). Also, I noticed that `toFixed` will always return a number that has the specified number of decimal places (so 2 will come out as "2.0"). I like how in the designs extra zeroes are chopped off of the number. [This](https://github.com/MetaMask/metamask-extension/pull/13044/files#diff-d4258199a39e1cde63a08ca933ad7aed8d08dd186334e1665573d8f8dd560771R8) is what I ended up doing for the priority fee range, what do you think?

Reply:
I have already updated to it this morning 😅 it took me a while to push the changes